CHANGELOG

```diff
@@ -1,4 +1,14 @@
 # Changelog
+## [0.2.2] - 2019-02-03
+### New Features
+ - Team interface support
+
+### Bug Fixes
+ - Fix memory leak of NM.Client
+ - Get SR-IOV VFs configuration from runtime
```

Comment:
`from kernel instead of reading the NetworkManager configuration`

Reply:
Done, thanks!